lib/src/configuration.cc

```
@@ -354,8 +358,7 @@ void Configuration::parseConfigFile() {
 
     // TODO(ale): update this for CTH-380
     if (!lth_file::file_readable(config_file_)) {
-        throw Configuration::Error { "config file '" + config_file_
```

If you agree, you can throw a specific error.